exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestPromotableWriter.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
All license headers should be already fixed as part of @ilooner PR #1207 or #1215.